### PR TITLE
Match nav button colors to theme

### PR DESCRIPTION
**Pull request by michaeljvaldes**
Are these style changes too gross? I figure if anywhere is gonna abuse inline styles, it's a component we won't reuse XD

### CODE REVIEW

**Comment by davidpchi** [comment]
So I think we should make the "on hover" style of the button be more bold. it looks like it is really faded.

**Comment by davidpchi** [comment]
Actually, the bigger point here is that the non-selected buttons in navigation look like they are disabled. They are too faded IMO. 

The actual selected button looks good though.

File: src/components/SidebarWithHeader.tsx [hunk 1]
```diff
@@ -165,9 +184,17 @@ const NavItem = ({ icon, children, route, onClose, ...rest }: NavItemProps) => {
...
                 role='group'
                 cursor='pointer'
+                color={isSelected() ? 'white' : 'black'}
+                bg={isSelected() ? 'primary' : 'white'}
+                opacity={isSelected() ? 1 : 0.5}
                 _hover={{
-                    bg: 'cyan.400',
+                    bg: 'primary',
                     color: 'white',
+                    opacity: isSelected() ? 1 : 0.5,
```
Comment (davidpchi) [inline src/components/SidebarWithHeader.tsx]: i don't think this opacity should be set. check with @nsthi  though but i'm not a fan. should just be "1" and never "0.5"

Comment (michaeljvaldes) [reply src/components/SidebarWithHeader.tsx]: Updated the buttons with a colorScheme which should control the behavior of the buttons a little better.

**Pull request merged**